src/2025h1/verification-and-mirroring.md

```diff
@@ -10,63 +10,101 @@
+We would like to have a experimental out-of-band version of a signing pipeline for the project for releases and crates.io. We expect this 6 month goal to consist of standing up experimental versions of the infrastructure and systems required for utilizing TUF on releases and crates with external commands and forks of the appropriate tools. This experimental version shall be a in-kind implementation of the RFC which is discussed in this goal. As a part of this process, we wish to take appropriate team time for discussion of the RFC for mirroring and signing to come to a consensus. The RFC shall be a transforming description ("living documentation") of the MVP implementation in order to drive discussion and show proof-of-concept to the appropriate teams.
+
+This goal shall include a series of educational materials (ex: blog posts, broken-down RFC components, or other materials) which discuss the history of artifact signing, current crate and release security, and the driving goals behind requiring cryptographic verification of Rust Project artifacts and why we have come to this solution. We hope to provide this material to interested parties across the project, with project-team-specific materials, to help drive consensus to this solution. These materials will not assume background knowledge of TUF or the TUF specification, and will provide motivation for the selection of TUF over other possibilities. These materials shall be crafted by the project team (@walterhpearce & @joshtriplett)
+
+We are requesting the following activities from Rust Project teams:
+- Leadership Council:
+  - Review of threat models, policy decision on whether those are the correct threat models to target, general approval about the use of a quorum to address those threat models.
+- Cargo Team:
+  - Review and approval of design for index changes and crate verification
+  - Review and approval of incremental bandwidth usage for updates using novel index update mechanism (to be created by @walterhpearce)
+- Crates.io Team:
+  - Review and approval of design for index changes
+  - Review and approval of rotation & revocation strategy
+- Infra Team:
+  - Review and approval of design for key management & ceremony process
+  - Review and approval of proposed repository structure
+  - Review and approval of secure storage & usage for automated keys
+- Release Team:
+  - Review of secure storage & usage for automated keys
+  - Review and approval of rotation & revocation strategy
```

Comment:
I'm not sure this is actually going to end up in T-release purview (vs. T-infra), because if we stick with the "release humans not involved" approach that release has been previously aiming for then release doesn't care. (There's some active discussion on this on the RFC thread; I think it's reasonable to leave this here and figure it out later. Maybe we could duplicate it into T-infra as a placeholder).